examples/with-jest-react-testing-library/package.json

```
@@ -0,0 +1,22 @@
+{
+  "name": "with-jest-react-testing-library",
+  "version": "1.0.0",
+  "dependencies": {
+    "next": "^7.0.2",
+    "react": "16.7.0-alpha.2",
```

Let's use stable React for the example, it doesn't seem like there is a strong requirement for React alpha:

```suggestion
     "react": "latest",
```